packages/block-library/src/comment-template/edit.js

```diff
@@ -103,7 +103,10 @@ function CommentTemplateInnerBlocks( {
 	);
 	return (
 		<li { ...innerBlocksProps }>
-			{ comment === ( activeComment || firstComment ) ? children : null }
+			{ comment.commentId ===
+			( activeComment?.commentId || firstComment?.commentId )
```

Comment:
I find it a bit uncommon to put the logical `||` on the RHS of the comparison (`===`). Can we maybe rewrite it as

```suggestion
			{ ( comment.commentId === activeComment?.commentId ) ||
			( ( comment.commentId === firstComment?.commentId )
```

or alternatively

```suggestion
			{ [ activeComment?.commentId, firstComment?.commentId ].includes ( comment.commentId )
```

Reply:
I've never heard of such a rule to be honest! Since they are logically equivalent so I just prefer the shorter version 🙂

But I'm happy to change it - I went with the version #1 from above.

Reply:
Oh, I should have been a bit more vigilant here because those are not actually logically equivalent 🤦 The original version expresses the correct logic whereas the "new" one does not. Let me explain.

Let's give the variables some values so let's say that:

```js
comment.commentId = 1;
activeComment?.commentId = 2,
firstComment?.commentId = 1;

// This is the "new" version
console.log(
  comment.commentId === activeComment?.commentId || 
  comment.commentId === firstComment?.commentId
); 
// true

// This is the "old" version
console.log(
  comment.commentId ===
    ( activeComment?.commentId || firstComment?.commentId )
); 
// false
```

In the old version, we short-circuit the RHS! This translates to:

If `activeComment?.commentId` is truthy (meaning that an active comment exists) compare it with `comment.commentId`. Otherwise, if the active comment does not exist, try to use the firstComment and compare it with `comment.commentId`.

Reply:
Ah, gotcha 🤔  Sorry for the accidental breakage introduced by my request! 😅